api/src/main/java/io/strimzi/api/kafka/model/KafkaBridgeSpec.java

```diff
@@ -68,14 +68,13 @@ public void setReplicas(int replicas) {
     }
 
     @JsonInclude(JsonInclude.Include.NON_NULL)
-    @Description("**Currently not supported** The Prometheus JMX Exporter configuration. " +
-            "See {JMXExporter} for details of the structure of this configuration.")
-    public Map<String, Object> getMetrics() {
-        return metrics;
+    @Description("Enable the metrics for the Kafka Bridge.")
```

Comment:
Document the default

Reply:
I had "... HTTP and Kafka metrics ..." there (I forgot JVM) but Jakub suggested being more generic in order to avoid that if adding more we had to change the description as well. Are you suggesting to revert back having something like "Enable the HTTP, Kafka, and JVM related metrics for the Kafka Bridge."?

Reply:
Does it default to true or false? I'm saying we should include what the default is in the `@Description`.

Reply:
:-D I completely misunderstood what you meant! sorry! Yes it default to false, I will add it.